### PR TITLE
Code changes for PCM-1857 - Send recommendation while case creation

### PR DESCRIPTION
**Pull request by vrathee**
@gandhikeyur @renujhamtani @engineersamuel  Please Review.
You might need to merge these changes to the webpackized PCM branch.

PS: Its a work under review. I have commented on the JIRA https://projects.engineering.redhat.com/browse/PCM-1857 . Once Spenser gives his ack on the JIRA, we can merge this.

### CODE REVIEW

**Comment by gandhikeyur** [comment]
Looks good. Reviewed!!

**Comment by engineersamuel** [comment]
Reviewed!

File: app/cases/services/caseService.js [hunk 1]
```diff
@@ -541,16 +541,15 @@ angular.module('RedhatAccess.cases').constant('CASE_GROUPS', {
...
-            //         resourceId: rec.id
-            //       });
-            //   });
-            // }
+            if(RHAUtils.isNotEmpty(recommendations)) {
+                caseJSON.recommendations = {
+                    recommendation: []
+                };
+                recommendations.forEach(function (rec,index) {
+                    console.log(recommendations.length);
```
Comment (jtrantin) [inline app/cases/services/caseService.js]: Not sure how final this version is, but we probably don't want to have this log in the merged version.

**Comment by jtrantin** [comment]
Apart from the extra console.log all seems good!

**Comment by vrathee** [comment]
@renujhamtani I have pushed the latest changes for this JIRA. We can consider this final an merge if we don't get any more comments till EOD.

@jtrantin I have removed the console log.. It was added in one of the intermediate commits.

**Comment by jtrantin** [comment]
Looking good!

**Pull request merged**